README.md

```diff
@@ -9,31 +9,63 @@ view-git-file macro on any Confluence page.
 ## Marketplace
 
 You can install Git for Confluence by using the Universal Plugin Manager inside
-Confluence, or by visiting the [Git for Confluence marketplace entry][1]
+Confluence, or by visiting the
+[Git for Confluence marketplace entry](https://avisi-apps.gitbook.io/git-for-confluence/.md).
 
 ## Documentation
 
 If you want to learn how to configure Git for Confluence or use the Git for
-Confluence macro's, please refer tot the [Git for Confluence documentation][2]
+Confluence macro's, please refer tot the
+[Git for Confluence documentation](https://avisi-apps.gitbook.io/git-for-confluence/).
 
 # Formats
 
+The are three main classes of formats that the Git for Confluence add-on natively
+supports. These consist of Markdown, Image and PlantUML files. Showcase examples
+that demonstrate how these can be used and rendered in Confluence are available
+in the formats directory:
+```bash
+.
+├── README.md
+└── formats
+    ├── image
+    │   ├── graph.png
+    │   ├── logo.svg
+    │   ├── success.jpg
+    │   └── tv.webp
+    ├── markdown
+    │   ├── image-reference.md
+    │   └── images
+    │       └── success.jpg
+    └── plantuml
+        ├── multi-block.puml
+        ├── multi-page.puml
+        └── simple-sequence-diagram.puml
+```
+These files can be directly added to the ``view-git-file`` Macro in Confluence
+and rendered on a page.
+
 ## Image
 
 Several images in several formats are available in the formats/markdown folder.
-An images will simply be show as an image when included on a Cofluence page by
-using the view-git-file macro.
+An images will simply be show as an image when included on a Confluence page by
```

Comment:
An image _or_ Images

Reply:
I only fixed the misspelled Confluence there. Will fix that too!